app/lib/common/utils/utils.dart

```diff
@@ -178,19 +178,19 @@ String getHumanReadableFileSize(int bytes) {
 
 String documentTypeFromFileExtension(String fileExtension) {
   switch (fileExtension) {
-    case 'png':
-    case 'jpg':
-    case 'jpeg':
+    case '.png':
```

Comment:
Wouldn't this require to check if platform is windows? What about other platforms ? Are you sure this wouldn't break the other ones?

Reply:
I think `path` package will support all platforms including windows.